lib/cloudregister/smt.py

```diff
@@ -40,9 +41,11 @@ def __init__(self, smtXMLNode, https_only=False):
         self._fqdn = smtXMLNode.attrib['SMTserverName']
         self._fingerprint = smtXMLNode.attrib['fingerprint']
         self._cert = None
+        self._cert_names = ('smt.crt', 'rmt.crt')
         self._protocol = 'http'
         if https_only:
             self._protocol = 'https'
+        self._form_srv_check_urls()
```

Comment:
Consider doing the assignment to `self._check_urls` here, consistent with the other assignments, instead of inside the `self._form_srv_check_urls()` method.

Reply:
e.g. ending up with

```
self._check_urls = self._form_srv_check_urls()
```

Reply:
+1 or consider renaming the method to `self._set_check_urls()`